src/js/angular/core/directives/dynamic-configuration-form/templates/dynamic-configuration-form.html

```diff
@@ -0,0 +1,44 @@
+<div class="dynamic-configuration-form">
```

Comment:
What's the reason for not having a `form` around these fields?

Reply:
added